data/completion/snap

```diff
@@ -38,15 +38,20 @@ _complete_snap() {
         fi
     done
 
-    command=${words[1]}
+    command="${words[1]}"
 
     # Only split on newlines
     local IFS=$'\n'
 
     # now we pass _just the bit that's being completed_ of the command
     # to snap for it to figure it out. go-flags isn't smart enough to
     # look at COMP_WORDS etc. itself.
-    COMPREPLY=($(GO_FLAGS_COMPLETION=1 snap "$command" "$cur"))
+    if [ "$command" = "debug" ]; then
+        command="${words[2]}"
```

Comment:
Do you need anything like `${foo:-}` in case `foo` is undefined?

Reply:
After writing and debugging and testing the change, I realised that we already checked for that (and bail out early)! So, no.